test/checkArrayExpectation.js

```diff
@@ -12,11 +12,12 @@ module.exports = function checkArrayExpectation(testDirectory, object, kind, fil
 		return !/from UglifyJs/.test(item);
 	});
 	if(fs.existsSync(path.join(testDirectory, filename + ".js"))) {
-		var expected = require(path.join(testDirectory, filename + ".js"));
+		var expectedFilename = path.join(testDirectory, filename + ".js");
+		var expected = require(expectedFilename);
 		if(expected.length < array.length)
-			return done(new Error("More " + kind + "s while compiling than expected:\n\n" + array.join("\n\n"))), true;
+			return done(new Error("More " + kind + "s while compiling than expected:\n\n" + array.join("\n\n") + ". Check expected warnings: " + filename)), true;
```

Comment:
Did you want to show `expectedFilename` instead of `filename` here? This way is only displays `warnings`.